jardin/database/base.py

```diff
@@ -26,6 +26,7 @@ def connection(self):
                 conn.commit()
         except self.DRIVER.InterfaceError:
             self._connection = None
+            self._cursor = None
```

Comment:
Not sure if this is the right fix as I was simply reading the previous commit before pool change and noticed this difference: https://github.com/instacart/jardin/blob/505b1ba294f949bc3d4f6c87fd3d7fdc81c89150/jardin/database/base.py#L59